app/views/layouts/application.html.erb

```diff
@@ -9,20 +9,22 @@
 <body>
 
 
-<div class="top-bar" id="responsive-menu">
+<div class="top-bar sticky" data-sticky data-margin-top="0" >
   <div class="top-bar-left">
-    <h1 class="menu-text">Zero to Hero</h1>
-
-<a href="/new">Add a Hero</a>
-<a href="/superheroes">All Superheroes</a>
+<a href="/new" class="__item">Add a Hero</a>
+<a href="/superheroes" class="__item">All Heroes</a>
+<a href="#" class="__item">My Account</a>
   </div>
+  <div class="__title">
+     <h1 >Zero to Hero</h1>
+</div>
```

Comment:
The indentation from 14-20 needs to be cleaned up to match the divs they are nested within